### DISCUSSION TITLE
Starred Repositories Still Show Up in Starred List after Adding to List

### DISCUSSION THREAD

**Discussion by LiuJiLan** [community/community#51199] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Recently, GitHub introduced a new feature called "List," which allows users to group repositories into different lists for better organization and management of their favorites. This is a very useful feature, but I have found that it does not fully achieve the goal of organizing starred repositories.

I noticed that even after I add a repository to one of my lists, it still shows up in my starred list. This means that I still have to browse through a lot of favorites in my starred list, which can easily be overwhelming. I think this completely defeats the purpose of adding to a list, as the lists should be helping us better organize and manage our starred list.

My suggestion is for GitHub to allow us to remove repositories from the starred list once they have been added to a list. This way, we can better utilize lists to organize our favorites and reduce unnecessary clutter in our starred list.

In summary, I think that starred repositories still showing up in the starred list after adding them to a list limits our ability to utilize lists for organization and management. I hope that GitHub can fix this issue soon so that we can better utilize this new feature.

**Comment by davidsonff** [comment]
Yes, or, alternatively, just be able to sort the starred repos by list with a grouping for "No List Selected" or something.
